app/views/roasts/index.html.erb

```diff
@@ -0,0 +1,70 @@
+<%= render partial: "shared/partials/header-global", locals: {title: "The right roast is out there.", text: "But with so many different types and brands, it can be hard to find. Let us suggest the perfect roast. Slide the thingy."} %>
```

Comment:
I changed the functionality on header-global so that I can generate it this way but I'll change it back